.gitlab-ci/packet.yml

```diff
@@ -73,6 +73,20 @@ packet_cleanup_old:
     - make cleanup-packet
   after_script: []
 
+packet_centos7-calico-ha-once-localhost:
```

Comment:
can remove this  job

Reply:
This commit is only in the test; I will remove it before merging.